src/preprocessing/passes/apply_substs.cpp

```diff
@@ -60,7 +60,7 @@ PreprocessingPassResult ApplySubsts::applyInternal(
       assertionsToPreprocess->replace(
           i,
           theory::Rewriter::rewrite(
-              assertionsToPreprocess->getTopLevelSubstitutions().apply(
+              d_preprocContext->getTopLevelSubstitutions().apply(
```

Comment:
Store `d_preprocContext->getTopLevelSubstitutions()`  in a variable before the loop and access it here.

Reply:
Done